common/src/main/java/whocraft/tardis_refined/common/block/device/ConsoleConfigurationBlock.java

```diff
@@ -89,10 +94,26 @@ public InteractionResult use(BlockState blockState, Level level, BlockPos blockP
         BlockState consoleBlock = level.getBlockState(blockPos.offset(offset));
         ConsoleTheme nextTheme = blockState.getValue(ConsoleConfigurationBlock.CONSOLE).next();
 
+        if(player.getMainHandItem().getItem() == ItemRegistry.PATTERN_MANIPULATOR.get()){
+
+            if (level.getBlockEntity(blockPos.offset(offset)) instanceof GlobalConsoleBlockEntity globalConsoleBlock) {
+                ConsoleTheme console = globalConsoleBlock.getBlockState().getValue(GlobalConsoleBlock.CONSOLE);
+                globalConsoleBlock.setPattern(ConsolePatterns.next(console, globalConsoleBlock.pattern()));
```

Comment:
Could be good to incorporate the "fail" sound that is used for controls if there is no patterns to cycle through. That way the player knows for sure that there's only one and not server-side lag or a "bug that doesn't let them swap all consoles".

Its a note sound that can be found in the controls or in the landing pad.

Reply:
Fixed